config/locales/en.yml

```diff
@@ -2900,6 +2912,23 @@ en:
       showing_page: "Page %{page}"
       next: "Next »"
       previous: "« Previous"
+  user_mutes:
+    index:
+      you_have_muted_n_users:
+        zero: "You haven't muted any Users 🎉"
+        one: "You have muted %{count} User"
+        other: "You have muted %{count} Users"
+      user_mute_explainer: "Private messages of muted users are moved into a separate inbox and you won't receive email notifications."
+      user_mute_admins_and_moderators: "You can mute Admins and Moderators but their private messages cannot."
```

Comment:
```suggestion
      user_mute_admins_and_moderators: "You can mute Admins and Moderators but their private messages cannot be muted."
```

Reply:
That change is of course equivalent to saying "you can mute them but it has no effect" so should we just not allow them to be muted?

I guess if their status changes later then the mute would suddenly activate but then that raises more questions...

Reply:
> That change is of course equivalent to saying "you can mute them but it has no effect" so should we just not allow them to be muted?

right. Users are able to mute Admins/Moderators but muting doesn't have an effect.

Reply:
> so should we just not allow them to be muted?

🤦🏽 I rushed over your text. You mean that we could just not display the `Mute this user`-action!

> I guess if their status changes later then the mute would suddenly activate but then that raises more questions...

Exactly this.